python/ray/util/placement_group.py

```diff
@@ -33,9 +52,8 @@ def ready(self) -> ObjectRef:
         """
         self._fill_bundle_cache_if_needed()
 
-        @ray.remote(num_cpus=0, max_calls=0)
-        def bundle_reservation_check(placement_group):
-            return placement_group
+        global bundle_reservation_check
```

Comment:
Is this redundant?

Reply:
Is it implicitly captured?

Reply:
![image](https://user-images.githubusercontent.com/13081808/94871404-775d6f80-047c-11eb-93a3-f9e59f38937c.png)
We only need to make a statement where it's updated?